### PR TITLE
[BugFix][TVMScript] Fix the roundtripability of  intrinsic pow

### PR DESCRIPTION
**Pull request by lightzhan-intellif**
Hi,
This is a bug fix discussed [here](https://github.com/apache/tvm/issues/13674). Any further suggestion is welcome.

cc @junrushao

### CODE REVIEW

**Comment by lightzhan-intellif** [comment]
@tvm-bot rerun

File: tests/python/unittest/test_tvmscript_roundtrip.py [hunk 1]
```diff
@@ -3622,5 +3622,14 @@ def test_return_none_no_trailing_type():
     assert "-> None" not in script
 
 
+def test_pow_roundtripable():
```
Comment (Hzfengsy) [inline tests/python/unittest/test_tvmscript_roundtrip.py:3625]: Could you please use the `ir_generator` to make tests consistent in this file?

**Review by junrushao — APPROVED** [review]
LGTM. Please address comments from @Hzfengsy

**Comment by junrushao** [comment]
BTW, there are some existing logics that depend on the python API `tir.power`, so please keep a copy as it is for backward compatibility

File: tests/python/unittest/test_tvmscript_roundtrip.py (hunk 1 above)
Comment (lightzhan-intellif) [reply tests/python/unittest/test_tvmscript_roundtrip.py:3625]: Okay.

**Comment by lightzhan-intellif** [comment]
Do you mean that users can use both `tir.pow` and `tir.power`?  I think it looks like a little bit confusing and redundant. How about we do not make any change to `tir.power`?

File: python/tvm/script/ir_builder/tir/ir.py [hunk 2]
```diff
@@ -1532,7 +1532,7 @@ def wrapped(*args, **kwargs):
 nearbyint = _op_wrapper(_tir_op.nearbyint)
 nextafter = _op_wrapper(_tir_op.nextafter)
 popcount = _op_wrapper(_tir_op.popcount)
-power = _op_wrapper(_tir_op.power)
+pow = _op_wrapper(_tir_op.power)  # pylint: disable=redefined-builtin
```
Comment (junrushao) [inline python/tvm/script/ir_builder/tir/ir.py:1535]: I was thinking if we could add a `pow` method to `tvm/tir/op.py` so that it could be consistent

Comment (lightzhan-intellif) [reply python/tvm/script/ir_builder/tir/ir.py]: Done.

**Review by Hzfengsy — APPROVED** [review]

**Pull request merged**